Pilot/vm-pilot.sh

```diff
@@ -0,0 +1,123 @@
+#!/bin/sh
+#
+# Runs as dirac. Sets up to run dirac-pilot.py
+#
+
+date --utc +"%Y-%m-%d %H:%M:%S %Z vm-pilot Start vm-pilot"
+
+#checking if stomp is installed
+if ! python -c 'import stomp' > /dev/null 2>&1; then
```

Comment:
I think that, in order to be more flexible, you may add a "requirements.txt" file that for the moment just include stomp. Then, here you just add "pip install -r requirements.txt". This same pattern is also in DIRAC (explained in http://dirac.readthedocs.io/en/rel-v6r15/DeveloperGuide/DevelopmentEnvironment/DeveloperInstallation/index.html#installing-the-dependencies) so in case there are more requirements in terms of packages we just need to add a line in this file.